test_scripts/Policies/user_consent_of_Policies/219_ATF_Device_identifier_creation_upon_connect_no_apps.lua

```diff
@@ -13,51 +13,86 @@
 -- Expected result:
 -- SDL must add new device in deviceList of BasicCommunication.UpdateDeviceList
 ---------------------------------------------------------------------------------------------
+--[[ General Settings for configuration ]]
+Test = require('user_modules/dummy_connecttest')
+
 --[[ Required Shared libraries ]]
 local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
 local commonSteps = require('user_modules/shared_testcases/commonSteps')
-local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
 local utils = require ('user_modules/utils')
+local hmi_values = require('user_modules/hmi_values')
+local SDL = require('SDL')
 
 --[[ General Precondition before ATF start ]]
-commonSteps:DeleteLogsFileAndPolicyTable()
-
---[[ General Settings for configuration ]]
-Test = require('user_modules/connecttest_resumption')
-require('cardinalities')
-require('user_modules/AppTypes')
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable()
 
---[[ Preconditions ]]
-commonFunctions:newTestCasesGroup("Preconditions")
+--[[ Local Variables ]]
+local hmiParams = hmi_values.getDefaultHMITable()
+local vin = hmiParams.VehicleInfo.GetVehicleData.params.vin
 
---[[ Test ]]
-commonFunctions:newTestCasesGroup("Test")
-function Test:Test_Connect_device()
-  commonTestCases:DelayedExp(2000)
-  self:connectMobile()
-  EXPECT_HMICALL("BasicCommunication.UpdateDeviceList",
-    {
-      deviceList = {
-        {
-          id = utils.getDeviceMAC(),
-          name = utils.getDeviceName(),
-          transportType = "WIFI",
-          isSDLAllowed = false
-        }
+--[[ Local Functions ]]
+function Test:initHMIonReady()
+  hmiParams.BasicCommunication.UpdateDeviceList = nil
+  EXPECT_HMICALL("BasicCommunication.UpdateDeviceList", {
+    deviceList = {
+      [1] = {
+        id = utils.buildDeviceMAC("WS", { vin = vin }),
```

Comment:
Since VIN was removed as a dependency should this be changed?

Reply:
@JackLivio Please find update in c347120